src/style.css

```diff
@@ -1,3 +1,77 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap';
+
 body {
   margin: 0;
+  background-color: #e1e3e1;
+  font-family: Roboto, sans-serif, system-ui;
```

Comment:
It's good practice to be consistent with vertical margins. Decide on a direction (top or bottom) and stick to it to avoid potential margin collapsing issues. In this case, you have set both `margin: 0` on the `body` and `margin: 0` on the `header`. Consider using only one direction for vertical margins throughout your stylesheet.